components/brave_news/browser/publishers_controller.cc

```diff
@@ -188,11 +187,10 @@ void PublishersController::EnsurePublishersIsUpdating() {
     return;
   }
   is_update_in_progress_ = true;
-  std::string region_part = brave_news::GetRegionUrlPart();
   GURL sources_url("https://" + brave_news::GetHostname() + "/sources." +
-                   region_part + "json");
+                   brave_news::kRegionUrlPart + "json");
```

Comment:
This is ultimately fine - but we might want to do the string operations using something like `base::StringPrintf`()

Reply:
I think I'll use base::StrCat instead, if that's okay. I think it reads slightly nicer than the `StringPrintf` and Chromium doesn't like me passing a `std::string` into `StringPrintf` so I have to call `c_string()` on `GetHostname`, which is kinda ugly.

Reply:
This is way nicer! 😄👍